lib/sentry/context.ex

```diff
@@ -92,6 +260,16 @@ defmodule Sentry.Context do
     Process.put(@process_dictionary_key, new_context)
   end
 
+  @doc """
+  Returns the keys used to store context in the current Process's process
+  dictionary.
+
+  ## Example
+
+      iex> Sentry.Context.context_keys()
+      [:breadcrumbs, :tags, :user, :extra]
+  """
+  @spec context_keys() :: list(atom())
```

Comment:
So I'm not entirely sure why this function exists or if it's even accurate (since it's leaving out `@request_key`.). I didn't see a reference to it anywhere else in this project, and am unclear on why it would be used by an application consuming this library.

Regardless, I'm assuming I'm missing something, and so added documentation for it anyhow.